config/v1alpha1/types_cluster_monitoring.go

```diff
@@ -89,6 +89,30 @@ type ClusterMonitoringSpec struct {
 	// The current default value is `DefaultConfig`.
 	// +optional
 	AlertmanagerConfig AlertmanagerConfig `json:"alertmanagerConfig,omitempty,omitzero"`
+	// prometheusK8sConfig provides configuration options for the Prometheus instance, which is the pod running Prometheus in the cluster.
+	// By default, at least one Prometheus pod is deployed in the `openshift-monitoring` namespace to collect and store metrics for the platform.
```

Comment:
Is this only for configuring the default prometheus instance or does the configuration specified here apply to _all_ prometheus instances?